assets/js.js

```diff
@@ -0,0 +1,210 @@
+            <li><button value=${gen.name} onClick="getGamesByGenres(this);" class="btn-game-type">${gen.name}</button></li>
+            `;
+            //lay value de tra ve gia tri khi click
+
+            if (temp < 7)
+                genresList.appendChild(x)
+        })
+
+    } catch (err) {
+        console.log("err", err)
+    }
+}
+//END - GET 6 GNERES
+renderGames()
+renderGenres()
+
+
+//BEGIN - GET GAMES BY NAME
+//get list games by search, genres and page
+const getGamesByName = async () => {
+    try {
+        const url = `https://cs-steam-game-api.herokuapp.com/games?q=${document.querySelector('#nav-search').value}&page=${document.querySelector('#nav-search-page').value}
+        `
+        //here is how we add a dynamic value (API KEY) to the url
+        const res = await fetch(url)
+        const gameByName = await res.json()
+
+        // console.log("gameByName", gameByName) //have a look the retrieved data
+        return gameByName
+    } catch (error) {
+        console.log('error', error)
+    }
+
+}
+const renderGamesSearch = async () => {
+    try {
+        errorMessage.style.display = 'flex'
+        const gameByName = await getGamesByName()
+        const gamesListByName = document.getElementById("content")
+        const ulGamesListByName = gamesListByName.children[1]
+        ulGamesListByName.innerHTML = ""
+
+        gameByName.data.forEach((game) => {
+
+            const x = document.createElement("div")
+            x.innerHTML = `
+            <div id=${game.appid} onClick="getGamesByClick(this)" class="game-wrraper">
+                <divclass="cover">
+                    <img  src="${game.header_image}"
+                        alt="">
+                    <div class="game-info">
+                        <p>${game.name}</p>
+                        <p>${game.price}$</p>
+                    </div>
+                </divclass=>
+            </div>    
+            `;
+            
+            ulGamesListByName.appendChild(x)
+        })
+        errorMessage.style.display = 'none';
+    } catch (err) {
+        console.log("err", err)
+    }
+}
+//END - GET GAMES BY NAME
+
+//BEGIN - GET GAMES BY GENRES
+const getGamesByGenres = async (obj) => {
+    // save value to nav-search
+    // console.log(obj.value)
+    document.querySelector('#nav-search').value = obj.value //get value in html element
+    renderGamesSearch()
+}
+//END - GET GAMES BY GENRES
+getGamesByName()
+const getGamesByPage = async (obj) => {
+    //create input element id='nav-search-page' and hide it (display: none)
+    document.querySelector('#nav-search-page').value = obj.value //get value in html element
+    renderGamesSearch()
+}
+
+searchBtn.addEventListener('click', () => {
+    renderGamesSearch()
+})
+
+//Click to see more info
+const getGamesByClick = async (obj) => {
+    // save value to nav-search-click
+    console.log(obj.id)
+    document.querySelector('#nav-search-click').value = obj.id
+    renderGamesByClick()
```

Comment:
```suggestion
    await renderGamesByClick();
```

Do you know what is `async/await` for?